pkg/storer/metrics.go

```diff
@@ -18,6 +18,12 @@ import (
 type metrics struct {
 	MethodCalls         prometheus.CounterVec
 	MethodCallsDuration prometheus.HistogramVec
+	ReserveSize         prometheus.Gauge
+	StorageRadius       prometheus.Gauge
+	CacheSize           prometheus.Gauge
+	EvictedCount        prometheus.Counter
```

Comment:
These names should be more descriptive, more specifically, to what the evicted and expired are referring to.

Reply:
The help mentions more details. This is how it looks on the metrics endpoint:

```
# HELP bee_localstore_evicted_count Number of chunks evicted from reserve.
# TYPE bee_localstore_evicted_count counter
bee_localstore_evicted_count 2.474865e+06
# HELP bee_localstore_expired_count Number of chunks expired from reserve due to stamp expirations.
# TYPE bee_localstore_expired_count counter
bee_localstore_expired_count 0
```

I think it should be clear right?